packages/web-components/src/utilities/type-ramp.ts

```diff
@@ -0,0 +1,12 @@
+/** @public */
+export enum StandardFontWeight {
+  Thin = 100,
+  ExtraLight = 200,
+  Light = 300,
+  Normal = 400,
+  Medium = 500,
+  SemiBold = 600,
+  Bold = 700,
+  ExtraBold = 800,
+  Black = 900,
```

Comment:
Just for being able to look back to this PR and see the reasoning, can you given some perspective on using a color for the final value here?

Reply:
I was trying to find the origin of the naming, I suspect historically the strongest weight used the most ink, and therefore was more "black". I copied over the standard names and weights from https://developer.mozilla.org/en-US/docs/Web/CSS/@font-face/font-weight#common_weight_name_mapping.